src/test/java/Models/TrainingProgramTest.java

```diff
@@ -0,0 +1,37 @@
+package Models;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Assert;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TrainingProgramTest {
```

Comment:
please add more testcases with calculations. you can make them parameterized tests to avoid duplication